freeCodeCamp/freeCodeCamp
# fix(client): remove react-bootstrap for user token widget

Noticed this while working on another PR. This replaces react-bootstrap with our component library. There should be no visual difference.

To test:
-go to a relational database project and click the "start" button. This will generate a user token
-go to settings page, scroll to bottom, see user token widget

Checklist:






Closes #XXXXX
